### PR TITLE
Tejas's View

### PR DESCRIPTION
**Pull request by tejas-cogo**
Mr Rahul Koshta ,Pls review my Code

### CODE REVIEW

File: app/views/layouts/tejas_view.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (Rkoshta79) [inline app/views/layouts/tejas_view.html:6]: hi tejas change the body

Comment (Rkoshta79) [reply app/views/layouts/tejas_view.html:6]: hi

**Review by tejas-cogo — COMMENTED** [review]
Sorry for disturbing pls check my changes

**Comment by Rkoshta79** [comment]
ghgfhghghh

File: app/views/layouts/tejas_view.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    <title>Document</title>
+</head>
+<body>
+   <h1>My First Github Project</h1>
```
Comment (Rkoshta79) [inline app/views/layouts/tejas_view.html]: change the name.

**Review by Rkoshta79 — CHANGES_REQUESTED** [review]
change the body.

**Pull request merged**